app/models/only_admins.rb

```diff
@@ -0,0 +1,5 @@
+class OnlyAdmins < ActiveAdmin::AuthorizationAdapter
+  def authorized?(action, subject = nil)
```

Comment:
Unused method argument - `action`. If it's necessary, use `_` or `_action` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.